scripts/stack.yaml

```diff
@@ -0,0 +1,16 @@
+##
```

Comment:
This is here because of `intero`, yes -- there is no `stack.yaml` that covers `scripts/aws.hs`.